content/using/install.md

```diff
@@ -82,9 +76,25 @@ Urbit cannot run on Windows itself, but there is a convenient way to run a Linux
 
 Once WSL 2 is installed, open a Linux terminal in Windows and then follow the Linux installation instructions above. For performance reasons, do not install Urbit in the mounted Windows volume, but install it in the Linux file system. For example, in your home directory, by typing `cd ~`.
 
-### Booting Your Comet
+## Launching Urbit
+
+Now that you have Urbit installed you can start it up with an ID.
+
+If this is your first time using Urbit, it'll be easiest to start with a comet in order to get on the network.
+
+### Comet: Install and Setup {#comet}
+
+If you don't have your own Planet ID, but still want to try out the Urbit network you can launch a comet.
+
+**Comets** are temporary 128-bit or 16 syllable Urbit IDs, that look like:
+
+`~dasres-ragnep-lislyt-ribpyl--mosnyx-bisdem-nidful-marzod`
+
+Comets are disposable, free identities that anyone can make to join the live network and try things out.
+
+### Booting a comet
 
-To boot a new comet, go into the command line and run the following command from the directory that was created during Urbit installation:
```

Comment:
extremely minor, but why was new removed?

Reply:
I don't think it was - I think this is a github diff bug.